package.json

```diff
@@ -9,7 +9,8 @@
   },
   "scripts": {
     "build": "grunt ; doctoc ./README.md ; doctoc ./doc/README.md ; doctoc ./doc/CHANGES.md",
-    "test": "grunt lint && karma start karma.conf.js"
+    "test": "grunt lint && karma start karma.conf.js",
+    "lint": "grunt lint"
```

Comment:
added an explicit lint (makes it easier to check before commit) as for some reason it's not auto linting in Visual Studio code yet...